sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/package-info.java

```diff
@@ -3,7 +3,35 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/** Samplers for selecting traces that are recorded and exported. */
+/**
+ * This package contains {@link io.opentelemetry.sdk.trace.samplers.Sampler}s for selecting traces
+ * that are recorded and exported. <br>
+ *
+ * <p>Sampling is a mechanism to control the noise and overhead introduced by OpenTelemetry by
```

Comment:
Should we reference specification here?

Reply:
Probably; I copied this line directly out of the spec. I'm slightly worried about javadoc referencing potentially mobile spec docs, but it might be fine.

Reply:
check it out. how stable do we think that URL is going to be?

Reply:
you can get a permanent link that points to a specific commit.

Reply:
That's an idea. Will that be something that we remember to update when the spec gets versioned?

Reply:
:)) everything has it's own problems

Reply:
switched to the latest SHA for now.